cmd/otelcol/main.go

```diff
@@ -154,18 +154,13 @@ func checkRuntimeParams() {
 		memTotalSizeMiB = val
 	}
```

Comment:
Can we add a comment near declaration of `ballastEnvVarName= "SPLUNK_BALLAST_SIZE_MIB"` that says that this env var name must be the same as what is used in the config files?

Reply:
That is true for all the env variables.